src/App.jsx

```diff
@@ -14,57 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectProduct, setSelectProduct] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleSelectGoods = good => {
+    setSelectProduct(good === selectProduct ? null : good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const GoodsList = ({ goodsList }) =>
+    goodsList.map(good => {
+      const isSelected = good === selectProduct; // Створено змінну для порівняння
```

Comment:
It's best to delete unnecessary comments to keep the code clean.

```suggestion
      const isSelected = good === selectProduct;
```